cypress/e2e/testing/loginlogout.cy.js

```diff
@@ -0,0 +1,64 @@
+const validEmail = "MyNameAgain24@noroff.no";
+const validPassword = "mynameagain24";
+
+const invalidEmail = "invalidEmail@noroff.no";
+const invalidPassword = "invalidPassword";
+
+describe("Login and logout test", () => {
+  beforeEach(() => {
+    cy.visit("/");
+    cy.clearLocalStorage();
+    cy.wait(1000);
+  });
+
+  it("Login with valid credentials", () => {
+    cy.get("#registerForm button[data-auth='login']").click();
+    cy.wait(1000);
+    cy.get("#loginEmail").type(validEmail);
+    cy.get("#loginPassword").type(validPassword);
+    cy.get("#loginForm button[type='submit']").click();
+    cy.wait(1000);
+    cy.get("button[data-visible='loggedIn']").should("exist");
```

Comment:
It would be nice to add more proof here, like checking the token

Reply:
I added a check for token now.